docs/operate/mobility/move-arm/configure-additional.md

```diff
@@ -18,25 +18,28 @@ If you have a gripper, camera, or other components attached to your arm, you can
 
 You can also take into account passive objects attached to the arm such as a camera mount to avoid collisions.
 
-## Configure a gripper
+## Configure a gripper's frame
 
 If you have a gripper, you can configure it to move with the arm.
-See [Configure a gripper](/operate/reference/components/gripper/) for instructions on configuring the gripper itself.
-Then, configure the gripper's frame to describe its position and orientation relative to the arm:
+Configure the gripper's frame to describe its position and orientation relative to the arm:
+
+1. See [Configure a gripper](/operate/reference/components/gripper/) for instructions on configuring the gripper itself.
```

Comment:
Just because of the change in the headline

```suggestion
If you haven't already configured your gripper, see [Configure a gripper].

Configure the gripper's frame to describe its position and orientation relative to the arm:
```